src/python/tests/test_cigar.py

```diff
@@ -0,0 +1,108 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of :mod:`cigar` module.
+
+The unit testing is divided into one test class per submodule/class found in this module, and one test method
+per public function/class method.
+
+Typical usage example::
+
+    $ pytest test_cigar.py
+
+"""
+
+from typing import Tuple, ContextManager, List
+from contextlib import nullcontext as does_not_raise
```

Comment:
Sorting the imports will make it easier to scan them and will help avoid the dreaded [warning about double imports](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/reimported.html).
```suggestion
from contextlib import nullcontext as does_not_raise
from typing import ContextManager, List, Tuple
```